lib/modules/keyboardNav.js

```diff
@@ -663,39 +663,39 @@ module.options = {
 		dependsOn: options => options.useGoMode.value,
 		callback() { if (module.options.useGoMode.value) toggleGoMode(); },
 	},
-	inbox: {
+	gotoInbox: {
```

Comment:
If you change option keys, you also need to add a migration to move the value from the old key to the new key.

Reply:
It may not be worth changing them in this PR.

Reply:
Okay, I'll revert that part.

Reply:
Any other ideas on how to best denote which options are go-mode enabled? Should I just group them better, all at the bottom of the list?